eth-account-snapshot/test/resolver.spec.js

```diff
@@ -1,62 +1,61 @@
+const expect = chai.expect
 const dagEthAccount = require('../index')
 const resolver = dagEthAccount.resolver
 const Account = require('ethereumjs-account')
 const emptyCodeHash = require('../../util/emptyCodeHash')
+const multicodec = require('multicodec')
 
 describe('IPLD format resolver (local)', () => {
-  let testBlob
-  let testData = {
-    nonce: new Buffer('02', 'hex'),
-    balance: new Buffer('04a817c800', 'hex'),
+  const testData = {
+    nonce: Buffer.from('02', 'hex'),
+    balance: Buffer.from('04a817c800', 'hex'),
     codeHash: emptyCodeHash,
-    stateRoot: new Buffer('012304a817c80004a817c80004a817c80004a817c80004a817c80004a817c800', 'hex')
+    stateRoot: Buffer.from('012304a817c80004a817c80004a817c80004a817c80004a817c80004a817c800', 'hex')
   }
-
-  before((done) => {
-    const testAccount = new Account(testData)
-    dagEthAccount.util.serialize(testAccount, (err, result) => {
-      if (err) return done(err)
-      testBlob = result
-      done()
-    })
-  })
+  const testAccount = new Account(testData)
+  const testBlob = dagEthAccount.util.serialize(testAccount)
 
   it('multicodec is eth-account-snapshot', () => {
-    expect(resolver.multicodec).to.equal('eth-account-snapshot')
+    expect(dagEthAccount.codec).to.equal(multicodec.ETH_ACCOUNT_SNAPSHOT)
   })
 
   it('defaultHashAlg is keccak-256', () => {
-    expect(resolver.defaultHashAlg).to.equal('keccak-256')
+    expect(dagEthAccount.defaultHashAlg).to.equal(multicodec.KECCAK_256)
   })
 
   describe('resolver.resolve', () => {
     it('path within scope', () => {
-      resolver.resolve(testBlob, 'nonce', (err, result) => {
-        expect(err).to.not.exist
-        expect(result.value.toString('hex')).to.equal(testData.nonce.toString('hex'))
-      })
+      const result = resolver.resolve(testBlob, 'nonce')
+      expect(result.value).to.eql(testData.nonce)
     })
+  })
 
-    it('resolves empty code', () => {
-      resolver.resolve(testBlob, 'code', (err, result) => {
-        expect(err).to.not.exist
-        expect(result.remainderPath).to.equal('')
-        expect(Buffer.isBuffer(result.value)).to.equal(true)
-        expect(result.value.length).to.equal(0)
-      })
-    })
+  it('resolves empty code', () => {
+    const result = resolver.resolve(testBlob, 'code')
+    expect(result.remainderPath).to.equal('')
+    expect(Buffer.isBuffer(result.value)).to.be.true()
+    expect(result.value.length).to.equal(0)
   })
 
   describe('resolver.tree', () => {
-    it('basic sanity test', () => {
-      resolver.tree(testBlob, (err, paths) => {
-        expect(err).to.not.exist
-        expect(Array.isArray(paths)).to.eql(true)
-      })
+    it('basic sanity test', async () => {
+      const tree = resolver.tree(testBlob)
+      const paths = [...tree]
+      expect(paths).to.have.members([
+        'storage',
+        'code',
+        'stateRoot',
+        'codeHash',
+        'nonce',
+        'balance',
+        'isEmpty',
```

Comment:
Would it be a big deal to have `resolve()` calls for each of these to properly sanity check? I suspect we'll find these two `is*()` calls resolve to `function`s.